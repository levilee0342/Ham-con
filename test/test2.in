-17051
-3521
-4355
-21232
867
1659
1005
-4972
-4073
-13352
-5718
-28385
-25743
-20457
