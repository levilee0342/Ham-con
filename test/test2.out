1659
-28385